src/pipecat/services/groq.py

```diff
@@ -64,17 +66,35 @@ def __init__(
         api_key: Optional[str] = None,
         base_url: str = "https://api.groq.com/openai/v1",
         language: Optional[Language] = Language.EN,
+        prompt: Optional[str] = None,
+        temperature: float = 0.0,
         **kwargs,
     ):
         super().__init__(
-            model=model, api_key=api_key, base_url=base_url, language=language, **kwargs
+            model=model,
+            api_key=api_key,
+            base_url=base_url,
+            language=language,
+            prompt=prompt,
+            temperature=temperature,
+            **kwargs,
         )
 
     async def _transcribe(self, audio: bytes) -> Transcription:
         assert self._language is not None  # Assigned in the BaseWhisperSTTService class
-        return await self._client.audio.transcriptions.create(
-            file=("audio.wav", audio, "audio/wav"),
-            model=self.model_name,
-            response_format="json",
-            language=self._language,
-        )
+
+        # Build kwargs dict with only set parameters
+        kwargs = {
+            "file": ("audio.wav", audio, "audio/wav"),
+            "model": self.model_name,
+            "response_format": "json",
+            "language": self._language,
+        }
+
+        if self._prompt is not None:
+            kwargs["prompt"] = self._prompt
+
+        if self._temperature is not None:
```

Comment:
`_temperature` is initialized to 0.0. Should it be optional in the constructor? If it's 0.0 do we want to actually use it?